src/main/java/com/example/demo/repository/cafe/BusinessHourQueryDslRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.repository.cafe;
+
+import static com.example.demo.domain.cafe.QBusinessHour.*;
+
+import java.time.DayOfWeek;
+
+import org.springframework.stereotype.Repository;
+
+import com.example.demo.domain.cafe.BusinessHour;
+import com.example.demo.domain.cafe.Cafe;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+
+import lombok.RequiredArgsConstructor;
+
+@Repository
+@RequiredArgsConstructor
+public class BusinessHourQueryDslRepository {
+	private final JPAQueryFactory jpaQueryFactory;
+
+	public BusinessHour findWithCafeAndDayOfWeek(Cafe cafe, DayOfWeek dayOfWeek) {
+		return jpaQueryFactory.from(businessHour)
```

Comment:
```suggestion
		return jpaQueryFactory.selectFrom(businessHour)
```
실제 쿼리 순서대로 작성하는게 가독성이 더 좋을 것 같아요!

Reply:
네 반영하겠습니다!